crates/noirc_evaluator/src/ssa/optimizations.rs

```diff
@@ -79,11 +79,15 @@ fn evaluate_intrinsic(
     block_id: BlockId,
 ) -> Result<Vec<NodeId>, RuntimeErrorKind> {
     match op {
-        builtin::Opcode::ToBits(_) => {
+        builtin::Opcode::ToBits(endian) => {
             let bit_count = args[1].to_u128() as u32;
             let mut result = Vec::new();
             let mut bits = args[0].bits();
             bits.reverse();
+            bits.resize(bit_count as usize, false);
```

Comment:
Looks like this changes the behavior a bit by removing the padding zeroes.

I'm alright with this, but it would be a breaking change for existing code.

Reply:
Good catch -- I think this is the correct behavior, since we want a u32 to return 4 bytes instead of 32.

Though as you noted, since this is a breaking change, we can put this in another PR/issue

Reply:
Looks like the code only creates stores to the resulting array for indices between 0 and the bit size. I think this PR is fine to merge then.